Remotion/Web/Core/Resources/ResourcePathBuilder.cs

```diff
@@ -40,7 +40,7 @@ public class ResourcePathBuilder : ResourcePathBuilderBase
 
     [UsedImplicitly]
     public ResourcePathBuilder (IHttpContextProvider httpContextProvider)
-      : this(httpContextProvider, WebConfiguration.Current.Resources.Root)
+      : this(httpContextProvider, SafeServiceLocator.Current.GetInstance<ResourceRoot>().Value)
```

Comment:
the ResourcePathBuilder already uses IoC, we can inject the new value via the ctor.

Reply:
I'm not sure if I did this right, if it's wrong, can you give me an example?

Reply:
I think I just did in the new review notes. the thread is separate so you might get this comment first?